src/khoj/main.py

```diff
@@ -126,6 +127,24 @@ def run(should_start_server=True):
     # Setup task scheduler
     poll_task_scheduler()
 
+    # Setup Background Scheduler
+    from django.conf import settings as django_settings
+
+    django_db = django_settings.DATABASES["default"]
+    state.scheduler = BackgroundScheduler(
+        {
+            "apscheduler.jobstores.default": {
+                "type": "sqlalchemy",
+                "url": f'postgresql://{django_db["USER"]}:{django_db["PASSWORD"]}@{django_db["HOST"]}:{django_db["PORT"]}/{django_db["NAME"]}',
```

Comment:
Can you explain how this works/what this does?

Reply:
APScheduler uses the postgres database (specified in `url`) as the job store. Job stores persist scheduled jobs in a DB table maintained by AP Scheduler. This code is pointing to the DB to use for maintaining scheduled job state by APScheduler